extensions/odoo_theme/layout_templates/legal.html

```diff
@@ -0,0 +1,150 @@
+<article id="o_content" class="doc-body o_legal">
+    <section id="legal">
+        <h1>
+            {{ _("Legal") }}
+            <a class="headerlink" href="#legal" title="Permalink to this headline">¶</a>
+        </h1>
+    </section>
+    <section id="licenses">
+        <h2>
+            {{ _("Licenses") }}
+            <a class="headerlink" href="#licenses" title="Permalink to this headline">¶</a>
+        </h2>
+        <ul class="simple">
+            <li>
+                <p>
+                    <a class="reference internal" href="legal/licenses.html#licenses">
+                        <span class="std std-ref">{{ _("Licenses") }}</span>
+                    </a>
+                </p>
+            </li>
+        </ul>
+    </section>
+    <section id="terms-and-conditions">
+        <h2>
+            {{ _("Terms and Conditions") }}
+            <a class="headerlink" href="#terms-and-conditions" title="Permalink to this headline">¶</a>
+        </h2>
+        <div class="row flex-wrap g-3 mb-3">
+            <div class="col-sm-4">
+                <div class="card h-100">
+                    <div class="card-body d-flex flex-column justify-content-between">
+                        <div class="d-flex align-items-center mb-3">
+                            <i class="i-article"></i>
+                            <h5 class="card-title mb-0">{{ _("Odoo Enterprise Agreement") }}
+                                <a class="footnote-reference brackets" href="#id2" id="id1">1</a>
+                            </h5>
+                        </div>
+                        <div>
+                            {%- set enterprise_agreement_path = 'legal/terms/i18n/enterprise_%s.html' % language_code %}
```

Comment:
The path for documents in English is `legal/terms/enterprise.html`. Clicking on Read in English leads to a 404.